css/css-responsive_playground/css/styles.css

```diff
@@ -62,7 +62,7 @@ h1 {
 .box--one {
   background-color: lightblue;
 
-  width: 100px;
```

Comment:
I'm not sure if we were supposted to cut the size in the half. I believe the % is right tough.

Reply:
You are definitely correct!